scripts/gis-material-update/README.md

```diff
@@ -323,6 +324,43 @@ Output files (names configured in `config.yaml`)
 
 - tormays_central_business_areas.gpkg
 
+### `liikennevaylat`
+
+Prerequisite: 
+- `central_business_area` material fetched
+- `ylre_katuosat` material fetched
+- `liikennevaylat` material fetched
+- `central_business_area` material processed
+- `ylre_katuosat` material processed
+
+In gis-material-update/.env file there need to be following secrets:
+
+```sh
+HELSINKI_EXTRANET_USERNAME=
+HELSINKI_EXTRANET_PASSWORD=
+```
+These secrets can be found from Azure Key vault or from OpenShift secrets (haitaton-secret).
```

Comment:
Maybe this kind of information should be in Confluence instead of public README.md file? I mean just the references to Azure and OpenShift might be considered as internal information not shared publicly?